templates/taxi/manufacturer_list.html

```diff
@@ -1,33 +1,34 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Manufacturer List
-    </h1>
-
-    {% if manufacturer_list %}
-      <table class="table">
+  <h1>Manufacturer list <a style="right: auto " href="{% url "taxi:manufacturer-create" %}">Create new one</a></h1>
+  {% if manufacturer_list %}
+    <table class="table">
+      <tr>
+        <th>ID</th>
+        <th>Name</th>
+        <th>Update</th>
+        <th>Delete</th>
+      </tr>
+      {% for manufacturer in manufacturer_list %}
         <tr>
-          <th>ID</th>
-          <th>Name</th>
-          <th>Country</th>
+          <td>
+            {{ manufacturer.id }}
+          </td>
+          <td>
+            {{ manufacturer.name }}
+          </td>
+          <td>
+            <a href="{% url "taxi:manufacturer-update" pk=manufacturer.id %}">UPDATE</a>
+          </td>
+          <td>
+            <a href="{% url "taxi:manufacturer-delete" pk=manufacturer.id %}">DELETE</a>
+          </td>
         </tr>
-
-        {% for manufacturer in manufacturer_list %}
-            <tr>
-              <td>
-                  {{ manufacturer.id }}
-              </td>
-              <td>
-                  {{ manufacturer.name }}
-              </td>
-              <td>
-                  {{ manufacturer.country }}
-              </td>
-          </tr>
-        {% endfor %}
-      </table>
-
-    {% else %}
-        <p>There are no manufacturers in the service.</p>
-    {% endif %}
+      {% endfor %}
+    </table>
+  {% else %}
+    <p>There are no manufacturer in the service</p>
```

Comment:
The message 'There are no manufacturer in the service' should be corrected to 'There are no manufacturers in the service' for proper grammar.